### PR TITLE
Handle null names more gracefully

### PR DESCRIPTION
**Pull request by mtompset**

### CODE REVIEW

File: SilMock/Google/Service/Directory/UsersResource.php [hunk 1]
```diff
@@ -356,12 +356,14 @@ public function listUsers($parameters = [])
             $userEntry = json_decode($userRecord['data'], true);
             if ($this->doesUserMatch($userEntry, $parameters['query'])) {
                 /** @var \Google_Service_Directory_UserName $newName */
+                $nameEntry = $userEntry['name'] ?? null;
+                $givenName = $nameEntry['givenName'] ?? null;
+                $familyName = $nameEntry['familyName'] ?? null;
+                $fullName = $nameEntry['fullName'] ?? trim($givenName . ' ' . $familyName);
```
Comment (jason-jackson) [inline SilMock/Google/Service/Directory/UsersResource.php:362]: What if givenName or familyName are null?

**Review by jason-jackson — COMMENTED** [review]
I think it still possibly errors out, but looks good.

File: SilMock/Google/Service/Directory/UsersResource.php (hunk 1 above)
Comment (briskt) [reply SilMock/Google/Service/Directory/UsersResource.php:362]: Apparently using the concatenate operator on null doesn't throw an error, but treats it as an empty string. (I had to test this to know.)

**Review by briskt — APPROVED** [review]

File: SilMock/Google/Service/Directory/UsersResource.php (hunk 1 above)
Comment (jason-jackson) [reply SilMock/Google/Service/Directory/UsersResource.php:362]: Thanks, I tried it online and I got an error, but maybe it didn't like how I tried it.

Comment (mtompset) [reply SilMock/Google/Service/Directory/UsersResource.php:362]: string coalesce for the win. ;)

**Pull request merged**